addon/helpers/moment-same.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+export function momentSame([date1, date2, unit]/*, hash*/) {
+  if (!date1 || !date2) {
```

Comment:
I'm not sure if I like this guard, maybe better to add an `Ember.assert` checking that this helper is only used with 2 dates? Also, this test is throwing an error because we're not using `moment`. Also also - it would be good to delete all the `moment-same-decade` and `moment-same-month` JS, HBS, and test files (and replace where these helpers are used in other tests).